edit-post/components/header/publish-button/index.js

```diff
@@ -0,0 +1,25 @@
+import { PostPublishPanelToggle, PostPublishButton } from '@wordpress/editor';
+
+const PublishButton = function( {
```

Comment:
Any reason for using `function() {` over `() => {`? We just tend toward fat-arrow functions in the codebase, so it stood out.

Reply:
4acd0c5086ae52408b5fcfe8a96d6dfe754183cc